seeds/endpointType.js

```diff
@@ -26,7 +26,7 @@
 
 const endpointTypes = [
   {
-    'name': 'FSIOP_CALLBACK_URL',
+    'name': 'FSPIOP_CALLBACK_URL',
```

Comment:
This should be removed as its no longer applicable. The [FSPIOP_CALLBACK_URL_TRANSFER_POST, FSPIOP_CALLBACK_URL_TRANSFER_PUT, FSPIOP_CALLBACK_URL_TRANSFER_ERROR] replaces this now.

Reply:
removed